src/main/java/org/ladocuploader/app/utils/HouseholdUtilities.java

```diff
@@ -1,34 +1,82 @@
 package org.ladocuploader.app.utils;
 
 import formflow.library.data.Submission;
-import java.util.ArrayList;
-import java.util.Calendar;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
+
+import java.util.*;
 
 public class HouseholdUtilities {
 
-  public static boolean isMember18orOlder(int day, int month, int year) throws NumberFormatException {
+  public static List<String> unmarriedStatuses = Arrays.asList("NeverMarried", "LegallySeparated", "Divorced", "Widowed");
+
+  public static Calendar ECE_CUTOFF_DATE = new Calendar.Builder().setDate(2019, 9, 30).build();
+  /**
+   * Checks the birthdate against the current calendar date (now) to see if the birthdate is 18 years or older.
+   * @param day day of Birth
+   * @param month month of Birth
+   * @param year year of Birth
+   * @return boolean - true if member is 18 or older, otherwise false
+   * @throws NumberFormatException throws this if the day, month or year passed in are invalid
+   */
+  public static boolean isMember18orOlder(int year, int month, int day) throws NumberFormatException {
+    return isMember18orOlder(year, month, day, Calendar.getInstance());
+  }
+  /**
+   * Checks the birthdate against the calendar date passed in to see if the birthdate is 18 years old or older.
+   *
+   * @param day day of Birth
+   * @param month month of Birth
+   * @param year year of Birth
+   * @param calendar calendar to compare the birthdate to
+   * @return boolean - true if member is 18 or older, otherwise false
+   * @throws NumberFormatException throws this if the day, month or year passed in are invalid
+   */
+  public static boolean isMember18orOlder(int year, int month, int day, Calendar calendar) throws NumberFormatException {
 
     if (day <= 0 || month <= 0 || year <= 0 ) {
       throw new NumberFormatException("cannot analyze birthdate as fields are missing");
     }
 
-    Calendar memberBirthDayCal = Calendar.getInstance();
-    memberBirthDayCal.set(Calendar.YEAR, year);
-    memberBirthDayCal.set(Calendar.MONTH, month);
-    memberBirthDayCal.set(Calendar.DAY_OF_MONTH, day);
+    Calendar memberBirthDayCal = new Calendar.Builder().setDate(year, month, day).build();
```

Comment:
`LocalDate` has "isBefore" and "isAfter" methods that might be more readable here.

Reply:
Pushed up a change based on this feedback